src/file_pipe_log/format.rs

```diff
@@ -19,6 +19,11 @@ const LOG_APPEND_SUFFIX: &str = ".raftlog";
 const LOG_REWRITE_SUFFIX: &str = ".rewrite";
 /// File header.
 const LOG_FILE_MAGIC_HEADER: &[u8] = b"RAFT-LOG-FILE-HEADER-9986AB3E47F320B394C8E84916EB0ED5";
+/// Mask of Version.
+const LOG_FILE_HEADER_VERSION_MASK: u64 = 0x00FFFFFFFFFFFFFF;
```

Comment:
Don't change the version field. Just add a new payload.

When reading the header, first call `let format = LogFileFormat::decode_version(peek(LogFileFormat::header_len())`, then call `format.decode_payload(peek(format.payload_len()))`.